css/index.css

```diff
@@ -71,10 +71,35 @@ p {
     margin: 0 auto;
 }
 
+
+
+header nav{
+    display: flex;
+	justify-content: space-between;
+	align-items: baseline;
+	margin-top: 25px;
```

Comment:
I would remove these margin-top margin-bottom and replace it with margin: 25px 0;